crates/neon/src/sys/async_work.rs

```diff
@@ -74,7 +74,7 @@ pub unsafe fn schedule<I, O, D>(
         Ok(()) => {}
         status => {
             // If queueing failed, delete the work to prevent a leak
-            napi::delete_async_work(env, *work);
+            napi::delete_async_work(env, *work).unwrap();
```

Comment:
Since this call isn't directly from the user and we are unwrapping a line later, I think it's best to ignore this return value.

We could consider adding a log warning, but I think it's okay for this to be best effort.